src/it/java/com/google/checkstyle/test/base/AbstractIndentationTestSupport.java

```diff
@@ -75,7 +75,7 @@ private static Integer[] getLinesWithWarnAndCheckComments(String aFileName,
                     throws IOException {
         final List<Integer> result = new ArrayList<>();
         try (BufferedReader br = Files.newBufferedReader(
-                Paths.get(aFileName), StandardCharsets.UTF_8)) {
+                Path.of(aFileName), StandardCharsets.UTF_8)) {
```

Comment:
https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/nio/file/Paths.html#get(java.net.URI)

>public static [Path](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/nio/file/Path.html) get([URI](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/net/URI.html) uri)
Converts the given URI to a [Path](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/nio/file/Path.html) object.
Implementation Requirements:
This method simply invokes [Path.of(URI)](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/nio/file/Path.html#of(java.net.URI)) with the given parameter.